### ISSUE TITLE
There is no way to disable identity verification (IDV) in the LMS

### ISSUE THREAD

**Issue by regisb** [openedx/build-test-release-wg#120]
In the LMS, when we setup a course in ecommerce with optionally verified seats, identity verification is mandatory. This means that students need to upload identity credentials documents before they can upgrade to a verified seat. This is a problem because many (most?) platforms don't use identity verification at all.

In practice, it means that when a user tries to upgrade, they are forwarded to the /id-verification url in the account MFE:

![upgrade](https://aws1.discourse-cdn.com/business7/uploads/openedx/optimized/2X/6/68f5f9d03dcb3dde006c0016330e3981840153ff_2_690x297.png)

I attempted to set the [`AUTOMATIC_VERIFY_STUDENT_IDENTITY_FOR_TESTING`](https://edx.readthedocs.io/projects/edx-platform-technical/en/latest/featuretoggles.html#featuretoggle-FEATURES['AUTOMATIC_VERIFY_STUDENT_IDENTITY_FOR_TESTING']) feature flag to bypass identity verification, but it not have any effect.

I attempted to manually bypass identity verification for a given user by going into the LMS admin, as suggested by Zia Zafal here: https://discuss.openedx.org/t/fixing-the-cybersource-issue-in-the-lilac-release-of-ecommerce/6044/4
Even if this solution was acceptable, it would not work; the following message is displayed to manually-validated users:

> You have already submitted your verification information. You will see a message on your dashboard when the verification process is complete (usually within 5 days).

@pshiu suggested [here](https://discuss.openedx.org/t/fixing-the-cybersource-issue-in-the-lilac-release-of-ecommerce/6044/8) that this issue may be caused by the fact that the course mode may be missing an SKU. Unfortunately I did not get the time to verify this.

This issue was first reported here: https://discuss.openedx.org/t/fixing-the-cybersource-issue-in-the-lilac-release-of-ecommerce/6044/3

**Comment by cmltaWt0** [comment]
Probably related post: https://discuss.openedx.org/t/deprecation-removal-id-verification-as-a-cert-requirement/6172

**Comment by ashultz0** [comment]
I think @pshiu is exactly right that having a verified seat SKU is the answer. I've gotten to this link myself locally and SKU'd my way to victory.

Without the IDV requirement the second IDV option for that link will go away entirely. Only the ecommerce branch remains, and the ecommerce needs a SKU to know what is being bought. If we hacked something into Maple to remove IDV from this page in advance of IDV requirement removal, we'd still need the SKU to talk with ecommerce.

One caveat: once you have a SKU you still get "you gotta verify before X date" messaging in a variety of places but you can go through with the purchase and run the course and so on. Disabling that messaging when IDV requirement is disabled is part of the work in process in master.

**Comment by regisb** [comment]
Thanks for investigating this issue @ashultz0. How did you run ecommerce? In the devstack, the native installation or with Tutor? How did you manage to configure the course SKU? IDV messages should be considered bugs on platforms that do not make use of the feature, so could you please post screenshots of the places where the messages appeared?

**Comment by ashultz0** [comment]
SKU
I was running a local devstack, it was a while ago so details are fuzzy. I believe I forced in a SKU for my test course using admin then because I was in a hurry, but Phil gave instructions on how to do it correctly in the linked discussion.

On "Verified" without IDV
In the current codebase verified = IDV. So if you're using verified seats, you are using IDV, they're not separable concepts in Maple and earlier. I just came on to edX this summer so I don't know all of the history but I have not seen any hint that they were ever separate concepts before this project to make IDV optional.

Messaging
There are a variety of places messages can occur. IDV messaging appears in the course listings, as a status panel on the dashboard, on the progress tab for accessing your certificates, on the cert itself, and in the course timeline both in the sidebar and in its tab.

To Sum Up
So verification messages on verified seats in this and older releases is working as designed but that design is changing as fast as we can change it. The changes are widespread in edx-platform and include other libraries like name affirmation and proctoring. Backporting them to maple would be a mess even if they were all done.

**Comment by pomegranited** [comment]
What is the meaning of using a verified seat without IDV? Is there some way to manually "verify" students instead? Does using a different course mode work around this issue?

**Comment by ashultz0** [comment]
@pomegranited that's a good question. Verified will require an honor-code agreement and at least on edx.org a purchase. Have a look at https://discuss.openedx.org/t/deprecation-removal-id-verification-as-a-cert-requirement/6172 for more info and a chance to comment.

**Comment by regisb** [comment]
> What is the meaning of using a verified seat without IDV? Is there some way to manually "verify" students instead? Does using a different course mode work around this issue?

Unless I'm mistaken (as I often am regarding ecommerce issues), verified seats are the only way to sell course access in Open edX. When IDV is disabled, the word "verified" loses its meaning and actually becomes a synonym of "commercial".

**Comment by regisb** [comment]
Closed by the recent fix in the tutor-ecommerce maple branch: https://github.com/overhangio/tutor-ecommerce/pull/18 Thanks to @pshiu!
